File: one_line/fizzbuzz.php
<?php for($i=1;$i<=100;$i++) echo ($i%3==0 xor $i%7==0) ? ($i%3==0 ? "FizzBuzz<br />" : $i."<br />"): ($i%3==0 ? "Fizz<br />" : "Buzz<br />"); 
if(2==2 xor 3==3){
	echo "true";
} else {
	echo "false"
}
?>

File: without%/fizzbuzz.py
def check_multi(n,p):
    while n >= p: n -= p
    return True if n == 0 else False
for i in range(100):
    if check_multi(i,3) and check_multi(i,7):
        print "FizzBuzz"
    else:
        print "Fizz" if check_multi(i,3) else ("Buzz" if check_multi(i,7) else i)
